Feature/add team variable to jira sprint enhanced (#186)
fivetran/dbt_jira: CHANGELOG.md

# dbt_jira v1.9.0

[PR #186](https://github.com/fivetran/dbt_jira/pull/186) includes the following updates:

## Feature Update
- Adds configurable table variable `Include Team Dimension in Sprint Enhanced` to control whether the `team` column is included in `jira__sprint_enhanced`. Variable is set to `true` by default. When set to `false`, the table rolls up to one row per sprint with all metrics aggregated at the sprint level. Only applicable if the `team` table is synced in your connector. 

> - **For dbt Core users**: Introduces `jira_sprint_enhanced_include_teams` variable to control whether the team dimension is included in `jira__sprint_enhanced`. This variable only applies when `jira_using_teams` is also enabled. [See the README on how to configure this variable](https://github.com/fivetran/dbt_jira?tab=readme-ov-file#disable-models-for-non-existent-sources).

Reviewer: I think you wanted this link (should exist after you merge the PR).
```suggestion
- **For dbt Core users**: Introduces `jira_sprint_enhanced_include_teams` variable to control whether the team dimension is included in `jira__sprint_enhanced`. This variable only applies when `jira_using_teams` is also enabled. [See the README on how to configure this variable](https://github.com/fivetran/dbt_jira#controlling-team-level-granularity-in-jira__sprint_enhanced).
```

# dbt_jira v1.9.0

[PR #186](https://github.com/fivetran/dbt_jira/pull/186) includes the following updates:

## Feature Update
- Adds configurable table variable `Include Team Dimension in Sprint Enhanced` to control whether the `team` column is included in `jira__sprint_enhanced`. Variable is set to `true` by default. When set to `false`, the table rolls up to one row per sprint with all metrics aggregated at the sprint level. Only applicable if the `team` table is synced in your connector. 

> - **For dbt Core users**: Introduces `jira_sprint_enhanced_include_teams` variable to control whether the team dimension is included in `jira__sprint_enhanced`. Set to `true` by default. When set to `false`, the model rolls up to one row per sprint with all metrics aggregated at the sprint level. This variable only applies when `jira_using_teams` is also enabled. [See the README on how to configure this variable](https://github.com/fivetran/dbt_jira?tab=readme-ov-file#disable-models-for-non-existent-sources).

Reviewer: We can simplify this a little since it's duplicative of the Feature Update section. 
```suggestion
- **For dbt Core users**: Introduces `jira_sprint_enhanced_include_teams` variable to control whether the team dimension is included in `jira__sprint_enhanced`. This variable only applies when `jira_using_teams` is also enabled. [See the README on how to configure this variable](https://github.com/fivetran/dbt_jira?tab=readme-ov-file#disable-models-for-non-existent-sources).
```

> 

Reviewer: Extra space. 
```suggestion
```
